### ISSUE TITLE
Version Warning

### ISSUE THREAD

**Issue by zaiquiriw** [discordjs/guide#1194]
### Is your feature request related to a problem? Please describe.

On the Initial Files page, the initialization of the Client is now incorrect in v14. The initial index file:
```js
// Require the necessary discord.js classes
const { Client, Intents } = require('discord.js');
const { token } = require('./config.json');

// Create a new client instance
const client = new Client({ intents: [Intents.FLAGS.GUILDS] });

// When the client is ready, run this code (only once)
client.once('ready', () => {
	console.log('Ready!');
});

// Login to Discord with your client's token
client.login(token);
```

### Describe the solution you'd like

While I think it would be cool to have all content affected by V14 could be changed to reflect that, it would make more sense to add a warning at the start of the guide until then. Warning we are using V13 for this tutorial, changes from V13 to V14 can be found [Here](https://discordjs.guide/additional-info/changes-in-v14.html#common-client-initialization-changes).

### Describe alternatives you've considered

For now, changing the initial file to:
```js
// Require the necessary discord.js classes
const { Client, GatewayIntentBits, Partials } = require('discord.js');
const { token } = require('./config.json');

// Create a new client instance
const client = new Client({ intents: [GatewayIntentBits.Guilds], partials: [Partials.Channel] });

// When the client is ready, run this code (only once)
client.once('ready', () => {
	console.log('Ready!');
});

// Login to Discord with your client's token
client.login(token);
```
would be helpful

### Additional notes

I'm just stepping into the guide, hope I didn't miss anything!

**Comment by zaiquiriw** [comment]
I forgot to note the page about updating from V13 to V14! That is useful, should have linked that in my suggestion ([update v14 guide](https://discordjs.guide/additional-info/changes-in-v14.html#enum-values))

**Comment by almeidx** [comment]
Fixed in #1146
